tasks/taxon_id/task_kraken2.wdl

```diff
@@ -52,6 +56,7 @@ task kraken2_theiacov {
     Float percent_sc2 = read_float("PERCENT_SC2")
     String percent_target_org = read_string("PERCENT_TARGET_ORG")
     String? kraken_target_org = target_org
+    File kraken2_classified_report = "~{samplename}.classifiedreads.txt.gz" 
```

Comment:
Having this floated to the data table as a gzipped text file feels odd to me. I got it confused with the regular report and couldn't figure out where to find the results. Recommendation to either (a) not output this file, or (b) not gzip it unless it's excessively big in which case i'd be more in favor of not outputting it if the reworked kraken2 report is sufficient

Reply:
This file is needed for the parsing task to recalculate abundances so I'm keeping it as one of the intermediate output files from Kraken2_standalone :) Thanks for the suggestion!